README.md

```diff
@@ -253,7 +253,7 @@ js-dir = "src"
 # A change to any file in those directories will trigger a rebuild.
 #
 # Optional.
-additionnal-watch-files = ["additionnal_files", "custom_config.json"]
+watch-additionnal-files = ["additionnal_files", "custom_config.json"]
```

Comment:
Can you please fix the typo throughout the PR, should be `additional` with a single N. Thanks!

Reply:
woups my bad, I'm not native english sorry